app/Dockerfile

```diff
@@ -8,11 +8,12 @@ ENV MM_VERSION=4.4.1
 ARG edition=entreprise
 
 # Install some needed packages
-RUN apt-get update \
-    && apt-get -y install \
+RUN apt-get update && \
```

Comment:
Please keep the `&&` part on the newline to be consistent with other Dockerfiles :)